src/Blockcore.Explorer/ClientApp/src/app/explorer/explorer.component.html

```diff
@@ -64,11 +64,15 @@ <h3>Network Statistics</h3>
             <span>Difficulty</span>
             <span>{{blockchain.difficulty}}</span>
          </div>
+         <div *ngIf="blockchain.networkWeight > 0">
```

Comment:
not sure if "blockchain" is just a basic model, and I know I fail on this myself all the time, but this really should be a get method that says: "hasNetworkWeight": boolean instead.

Reply:
A method where on the `ContractCodeComponent` class?

Reply:
lets fix this later